docker/database.mysql.yml

```diff
@@ -0,0 +1,21 @@
+# SQLite version 3.x
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+development:
+  adapter: mysql2
+  database: tsp_development
+  host: db
+  username: root
```

Comment:
Even if this is only an example config, I'd recommend to have a secure example. `Username: root` does not qualify as such ;-)

The same applies for Postgresql - please also use a non-privileged user in the example config there.

Reply:
This is just a development-oriented example. In such a case, you need to have privileges to create/destroy databases. However, I can try to come up with a more secure alternative.

Reply:
I don't even want to know how many "development-orientated examples" of various software are running in production... ;-)

Proposal: change it to something like
```
# needed MySQL permissions: full permissions (including create/drop database) for the database given above
- username: tsp-dev
```